appservice/src/github/connectToGitHub.ts

```diff
@@ -27,10 +27,10 @@ export type gitHubBranchData = { name: string };
 export type gitHubLink = { prev?: string, next?: string, last?: string, first?: string };
 export type gitHubWebResource = requestUtils.Request & { nextLink?: string };
 
-export async function connectToGitHub(node: AzureTreeItem, client: SiteClient, context: IConnectToGitHubWizardContext): Promise<void> {
+export async function connectToGitHub(subscriptionContext: ISubscriptionContext, client: SiteClient, context: IConnectToGitHubWizardContext): Promise<void> {
     const title: string = localize('connectGitHubRepo', 'Connect GitHub repository');
     context.client = client;
-    context.node = node;
+    context = Object.assign(context, { subscriptionContext });
```

Comment:
If you're gonna make `IConnectToGitHubWizardContext` extend `ISubscriptionContext`, there's no point in passing in the subscriptionContext separately. Typescript will enforce that the subscription stuff is on the GithubContext before it's passed in here

I'm guessing that's why you have another Object.assign above when you call `connectToGitHub`

Reply:
🤦